src/pages/Categories/CategoriesBeta.vue

```diff
@@ -10,7 +10,12 @@
 					future for themselves and their families.
 				</p>
 			</kv-grid>
-			<kv-grid class="md:tw-pt-6 lg:tw-pt-8">
+		</kv-page-container>
+		<div class="tw-pt-4 tw-mb-2">
```

Comment:
I think you can apply these classes directly on the loan-spotlight component and this wrapper div is unnecessary